samples/ComputerVision/AreaOfInterest/Program.cs

```diff
@@ -10,8 +10,8 @@ namespace Microsoft.Azure.CognitiveServices.Samples.ComputerVision.AreaOfInteres
 
     class Program
     {
-        public const string subscriptionKey = "<your training key here>"; //Insert your Cognitive Services subscription key here
-        public const string endpoint = "https://westus.api.cognitive.microsoft.com"; // You must use the same Azure region that you generated your subscription keys for.  Free trial subscription keys are generated in the westus region. 
+        public const string subscriptionKey = Environment.GetEnvironmentVariable("COMPUTER_VISION_SUBSCRIPTION_KEY"); //Insert your Cognitive Services subscription key here
+        public const string endpoint = Environment.GetEnvironmentVariable("COMPUTER_VISION_ENDPOINT"); // You must use the same Azure region that you generated your subscription keys for.  Free trial subscription keys are generated in the westus region. 
```

Comment:
@wiazur , I think the comment on this line is one you'd want to rewrite - it mentions region

Reply:
Thanks @PatrickFarley, did you remove that line? I see that Yuan merged it, and the merged file does not have that one. https://github.com/Azure-Samples/cognitive-services-dotnet-sdk-samples/blob/master/documentation-samples/quickstarts/ComputerVision/Program.cs

Reply:
The file you linked to is different from this one

Reply:
Ah, you're right. Sorry about that. I changed the comments. Good eye, thanks.
https://github.com/Azure-Samples/cognitive-services-dotnet-sdk-samples/pull/144